azerothcore/azerothcore-wotlk
# fix(DB/gameobject): Improve Hellfire Peninsula gathering nodes and pooling

## Changes Proposed:
First try using sub-zones
![image](https://user-images.githubusercontent.com/100873791/195465055-e98065cb-24f2-43a1-b63a-ca57cf95f8cc.png)

![image](https://user-images.githubusercontent.com/100873791/195464997-0e226154-ca41-42b2-b158-eca7282a04ec.png)

## Issues Addressed:

- Closes 

## SOURCE:

MangosDB for spawns. WotLK Classic for the rough quantities of each node

## Tests Performed:

- Tested briefly

## How to Test the Changes:


1. Preferrably download a gathering addon such as Gathermate
2. Fly around the zone gathering all the nodes you can.
3. See how many of each type spawn, see if the nodes respawn in different locations
4. See if Khorium spawns.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
